### PR TITLE
Upgrade lodash@3

### PR DESCRIPTION
**Pull request by ruimarinho**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "files": [
     "index.js",
     "lib",
-    "LECENSE"
+    "LICENSE"
```
Comment (sindresorhus) [inline package.json]: You can drop this btw. It's included by npm by default anyways.

**Comment by ruimarinho** [comment]
@sindresorhus done.

File: lib/template.js [hunk 2]
```diff
@@ -1,14 +1,16 @@
 var template = require('lodash.template');
-var reInterpolate = require('lodash._reinterpolate');
+var reEscape = require('lodash.reescape');
```
Comment (jdalton) [inline lib/template.js]: Sorry about this the exposed `lodash.reescape`, `lodash.reevaluate`, `lodash.reinterpolate` were a mistake. You can reference their former `lodash._reescape`, `lodash._reevaluate` `lodash._reinterpolate` forms.

Comment (contra) [inline lib/template.js]: @jdalton Want to PR really quick to correct that?

Comment (jdalton) [inline lib/template.js]: Yap, at work atm. Should be able to by noon.

Comment (jdalton) [inline lib/template.js]: Your code shouldn't break it just momentarily has an extra dep for each of the proxies regexp deps.

**Pull request merged**